packages/docusaurus-theme-live-codeblock/src/theme/Playground/index.tsx

```diff
@@ -81,6 +83,30 @@ function EditorWithHeader() {
   );
 }
 
+function ErrorFallbackWithHeader({error, tryAgain}: ErrorProps): JSX.Element {
```

Comment:
too much duplication here

Why not putting the error boundary inside ResultWithHeader instead? just wrap LivePreview?

Reply:
true

Reply:
![Capture](https://user-images.githubusercontent.com/64389512/187905572-6be327ba-c2e8-47db-b046-1265fbae6e6c.PNG)

wrapping `LivePreview` gives this behavior